data/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/Conversation.kt

```diff
@@ -328,6 +328,7 @@ sealed interface ConversationDetails {
         val selfRole: Conversation.Member.Role?
         val isFavorite: Boolean
         val folder: ConversationFolder?
+        val wireCell: String?
```

Comment:
Is this a name, an ID, or something else?

Would be nice to have a more descriptive name, or better yet, create a value class for it.